### DISCUSSION TITLE
How to Disable "Do you wish to see all X Possibilities?" prompt.

### DISCUSSION THREAD

**Discussion by Thy-GoD** [marlonrichert/zsh-autocomplete#625] [Q&A] [answered]
![image](https://github.com/marlonrichert/zsh-autocomplete/assets/123708601/82908fa4-b758-45e0-82a2-33d221d89a15)

Hello, I was just wondering how it'd be possible to disable this feature as I have tried before but couldn't find a way to do so, this keeps bugging me.

Thanks for looking at this thread!

**Comment by marlonrichert** [comment]
That's not a feature; it's a bug. 🙂

Can you please open a bug report for this and make sure you include a reproducible test case? Thanks!

**Reply by Thy-GoD** [reply]
Thats hilarious lol, what part of it is a bug and not a feature?

**Reply by marlonrichert** [reply, marked as answer]
Autocomplete should never show that message, but sometimes accidentally triggers it from Zsh.
